### PR TITLE
Implemented rudimentary user settings.

### PR DESCRIPTION
**Pull request by jasonirwin**
I Implemented rudimentary user settings. This update includes two settings for overriding of email TLD and email main domains. Please review and let me know if you have any questions/concerns.
Thank you
Jason

### CODE REVIEW

**Comment by kimpettersen** [comment]
@jasonirwin Thanks!,I'm excited about this!

Would you mind resolving the merge conflict?
The code looks good to me, but I would like to test it out manually before merging it. I'll try to get to that ASAP

**Comment by jasonirwin** [comment]
Done. As an aside, if user settings will be used elsewhere it might be an idea to create a superclass and instantiate in the constructor.

File: Random.sublime-commands [hunk 1]
```diff
@@ -11,6 +11,10 @@
     { "caption": "Random:Full Name", "command": "random_full_name" },
     { "caption": "Random:E-mail", "command": "random_email" },
     { "caption": "Random:Url", "command": "random_url" },
+<<<<<<< HEAD
```
Comment (kimpettersen) [inline Random.sublime-commands]: @jasonirwin Could you remove the conflict stuff?

**Comment by kimpettersen** [comment]
@jasonirwin Sorry for the late review, but I've tested it thoroughly now and once the merge conflict is solved I'm ready to merge it.

**Pull request closed without merging**